src/content/energy-consumption/index.md

```diff
@@ -7,50 +7,52 @@ sidebar: true
 
 # Ethereum energy consumption {#introduction}
 
-Ethereum's energy consumption will be reduced by ~99.95% following [The Merge](/en/upgrades/merge) from proof-of-work (PoW) to proof-of-stake (PoS). After The Merge, Ethereum will use dramatically less carbon to be more secure.
+Ethereum is a green blockchain. It uses a [proof-of-stake](/developers/docs/consensus-mechanisms/pos) consensus mechanism that
+can be run on low-powered devices and does not require heavy computation to participate. Under Ethereum's proof-of-stake mechanism
+the network is secured using staked Ether rather than expended energy (as in [proof-of-work](/developers/docs/consensus-mechanisms/pos)). This means the energy expended by the Etheruem network is very small - on the order of 0.01 TWhr/yr.
```

Comment:
Fix line length

```suggestion
Ethereum is a green blockchain. It uses a [proof-of-stake](/developers/docs/consensus-mechanisms/pos) consensus mechanism that can be run on low-powered devices and does not require heavy computation to participate. Under Ethereum's proof-of-stake mechanism the network is secured using staked Ether rather than expended energy (as in [proof-of-work](/developers/docs/consensus-mechanisms/pos)). This means the energy expended by the Etheruem network is very small - on the order of 0.01 TWhr/yr.
```